charts/backstage/values.yaml

```diff
@@ -101,6 +116,8 @@ upstream:
       # The initContainer below will install dynamic plugins in this volume mount.
       - name: dynamic-plugins-root
         mountPath: /opt/app-root/src/dynamic-plugins-root
+      - name: diststatic
+        mountPath: /opt/app-root/src/packages/app/dist/static
```

Comment:
This addition provides a needed ephemeral writeable volume since the root filesystem has been updated to be readonly